### PR TITLE
add overflow errors for int shifts.

### PR DESCRIPTION
**Pull request by alexpantyukhin**
fix for https://github.com/RustPython/RustPython/issues/556

### CODE REVIEW

File: vm/src/obj/objint.rs [hunk 1]
```diff
@@ -246,8 +246,7 @@ fn int_lshift(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     match get_value(i2) {
         ref v if *v < BigInt::zero() => Err(vm.new_value_error("negative shift count".to_string())),
         ref v if *v > BigInt::from(usize::max_value()) => {
-            // TODO: raise OverflowError
-            panic!("Failed converting {} to rust usize", get_value(i2));
+            Err(vm.new_overflow_error("the number is too large to convert to float".to_string()))
```
Comment (windelbouwman) [inline vm/src/obj/objint.rs]: Could you change the error message here?

**Comment by windelbouwman** [comment]
Hi @alexpantyukhin , thank you for your contribution to this project! I left one remark in your changes. Could you have a look at this?

File: vm/src/obj/objint.rs (hunk 1 above)
Comment (alexpantyukhin) [reply vm/src/obj/objint.rs]: ooops ) yes) `the number is too large to convert to int` would be ok?

**Comment by alexpantyukhin** [comment]
done! :)

**Pull request merged**